tests/travis/install.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+# NAME
+#     install.sh - Install Travis CI dependencies
+#
+# SYNOPSIS
+#     install.sh
+#
+# DESCRIPTION
+#     Creates the test fixture.
+
+cd "$(dirname "$0")"
+
+# Reuse ORCA's own includes.
+source ../../../orca/bin/travis/_includes.sh
+
+# If running our custom jobs or isolated test jobs, initialize the fixture.
+# Otherwise, use Orca's installation script.
+if [[ "$ACMS_JOB" == "base" ]] || [[ "$ACMS_JOB" == "starter" ]]; then
+  orca debug:packages CURRENT_DEV
+  orca fixture:init --force --sut=acquia/acquia_cms --sut-only --core=CURRENT_DEV --dev --profile=acquia_cms --no-sqlite --no-site-install
+  cat ../../patches/ci-settings.txt >> $ORCA_FIXTURE_DIR/docroot/sites/default/settings.php
+
+elif [[ "$ACMS_JOB" == "base_full" ]] || [[ "$ACMS_JOB" == "starter_full" ]]; then
+  orca debug:packages CURRENT_DEV
+  orca fixture:init --force --sut=acquia/acquia_cms --sut-only --core=CURRENT_DEV --dev --profile=acquia_cms --no-sqlite
+else
+# Run ORCA's standard installation script.
+  ../../../orca/bin/travis/install.sh
+fi
+
+printenv | grep ACMS_ | sort
+
+# If there is no fixture, there's nothing else for us to do.
+[[ -d "$ORCA_FIXTURE_DIR" ]] || exit 0
+
+cd $ORCA_FIXTURE_DIR
+
+# Rebuild cohesion after install.
+if [[ "$ACMS_JOB" != "base" ]] && [[ "$ACMS_JOB" != "starter" ]] && [[ "$ORCA_JOB" != "LOOSE_DEPRECATED_CODE_SCAN" ]] && [[ "$ORCA_JOB" != "DEPRECATED_CODE_SCAN_W_CONTRIB" ]] && [[ "$ORCA_JOB" != "STRICT_DEPRECATED_CODE_SCAN" ]]; then
+  drush cohesion:rebuild -y
+fi
+
+# Allow acquia_cms as allowed package dependencies, so that composer scaffolds acquia_cms files.
+composer config --json extra.drupal-scaffold.allowed-packages '["acquia/acquia_cms"]'
```

Comment:
This will replace the allowed-packages with `["acquia/acquia_cms"]`, shouldn’t  we use `--merge` option to keep the one provided by default if any?

Reply:
Yes. Make sense,